site/tutorials/snippets/09/install/istio.md

```diff
@@ -6,16 +6,16 @@ Download the Istio command line tool by [following the official instructions](ht
 
 <!-- command -->
 ```
-curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.10.0 sh -
+curl -L https://istio.io/downloadIstio | sh -
```

Comment:
This will always deploy the latest istio version, that could possible break the tutorial - so if a new istio version is released the next step will fail cause of the fixed path. Perhaps keeping the version would be a good idea.

Reply:
agree with Christian. Which version should we deploy then? is there a newer one than 1.10.0? or is that one good enought? @arthurpitman - thoughts on istio version?

Reply:
I used istio 1.11.2 with no problems, @checkelmann which version did you use?